tutorials/en/vectara-beginner-app-tutorial-showcase-the-creation-of-vectara-app-in-legal-use-case.mdx

```diff
@@ -0,0 +1,390 @@
+
+## Legal Consultation 📜
+
+The spotlight of this tutorial is a use case deeply rooted in the Legal domain. Amidst a burgeoning demand for swift and accessible legal consultations, this application emerges as a beacon of convenience. By harnessing the magic of automation and artificial intelligence, it offers preliminary legal advice derived from the uploaded documents, making legal assistance merely a click away.
+
+## Concept and Structure of the Application 🛠️
+
+The essence of this application is to provide a user-centric platform where obtaining legal consultations is a breeze with just a PDF document upload. The collaborative prowess of Streamlit, Vectara, and Langchain serves as the bedrock of this application, orchestrating a robust and intuitive environment.
+
+### Streamlit
+Streamlit, an open-source app framework, is the mastermind behind the interactive web interface of our application. It fosters a seamless user journey with widgets for file upload and elegantly unfurls the consultation output to the user, making the interface a joy to navigate.
+
+### Vectara 🧠
+Vectara is the linchpin that elevates the application's capabilities to a new zenith. As a semantic search software company, Vectara is on a mission to redefine search by leveraging artificial intelligence and neural network technologies for natural language processing. It facilitates a deeper understanding of user queries and provides extraordinarily relevant responses. In the scope of our application, Vectara processes the legal documents uploaded by users, delving into the semantics to extract crucial legal insights that form the basis of the automated consultations provided. The integration of Vectara transforms the application into a powerhouse of semantic search, ensuring users receive precise and relevant legal advice based on their documents.
+
+### Langchain 🤖
+Langchain, the engine of text generation in the application, sifts through the legal insights extracted by Vectara to generate automated legal advice. It is the cornerstone that enables the application to furnish text-based consultations, making legal advice easily accessible.
+
+The architecture of the application is elegantly simple yet potent. The user-friendly interface, sculpted with Streamlit, facilitates effortless uploading of PDF documents. Upon upload, Vectara springs into action, processing the document to extract legal insights. Langchain then takes the baton, generating legal advice that is promptly displayed to the user. This synergy ensures that users not only receive instant legal consultations but also have the option for further discussions with legal experts if needed.
```

Comment:
Suggestion for slight modification of "Langchain then takes the baton, generating legal advice that is promptly displayed to the user." to "Langchain then takes the baton, using Vectara's query API to retrieve relevant facts and then use those facts to generate legal advice that is promptly displayed to the user."